src/index.js

```diff
@@ -0,0 +1,145 @@
+let globalTemp;
+const cityInput = document.getElementById('name');
+const cityName = document.getElementById('city-name');
+const skyOptions = document.getElementById('skyOptions');
+const sky = document.getElementById('skyEmojis');
+
+//Wave 1
+const updateTemp = (temp) => {
+  const tempValueContainer = document.getElementById('tempValue');
+  tempValueContainer.textContent = temp;
+  globalTemp = temp;
+  updateTempColor(temp);
+  changeLandscape(temp);
+};
+
+const increaseTemp = () => {
+  globalTemp += 1;
+  updateTemp(globalTemp);
+};
+
+const decreaseTemp = () => {
+  globalTemp -= 1;
+  updateTemp(globalTemp);
+};
```

Comment:
These work great! Can you consider how you could make your code more DRY by combining them into a single function?